Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed issue leading to a stack overflow crash during device initialization in `InsertControlBitRangeNode` (case ISXB-405).
 - Fixed the issue where saving and loading override bindings to JSON would set unassigned overrides (that were `null`) to assigned overrides (as an empty string `""`).
 
+### Changed
+- Changed XR Layout build behavior to create Axis2D control devices with `StickControl` type instead of `Vector2Control`.
```

Comment:
This needs moved up to the Unreleased section. (Probably it was there to begin with, but got left behind after updating the branch).